docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -27,7 +27,7 @@ permalink: /FHIRServerUsersGuide/
   * [4.8 Using local references within request bundles](#48-using-local-references-within-request-bundles)
   * [4.9 Multi-tenancy](#49-multi-tenancy)
```

Comment:
Section 4.9.2.1 (line 1481)
"serviceClassName" : "com.ibm.fhir.audit.logging.impl.DisabledAuditLogService",
Replace with No-op.

Reply:
hum I don't see it ...

Reply:
Weird my local copy doesn't show this

Reply:
OH this is an old commit

Reply:
ok, thanks!  Not sure why I was seeing that.  Looks good now.